go/test/endtoend/vtgate/queries/derived/cte_test.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2022 The Vitess Authors.
```

Comment:
nit: i see this is a new file
```suggestion
Copyright 2023 The Vitess Authors.
```